clio-integration-test/src/main/python/clio_test.py

```diff
@@ -146,23 +147,43 @@ def testWithout(header, status):
 
 def test_read_group_metadata():
     id = str(uuid.uuid4()).replace('-', '')
-    library = 'library' + id
+    expected = {
+        'flowcell_barcode': 'barcode1',
+        'lane': 1,
+        'library_name': 'library' + id,
+        'project': 'testProject'
+    }
+    upsert = {'project' : expected['project']}
```

Comment:
unclear to me why this is an `upsert` operation, it looks like a regular `get` to me.